projects/angular/src/emphasis/_badges.clarity.scss

```diff
@@ -58,80 +40,30 @@
     vertical-align: middle;
     align-items: center;
     justify-content: center;
-    @include mixins.css-var(
-      min-width,
-      clr-badge-min-width,
-      variables.$clr_baselineRem_0_625,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      background-color,
-      clr-badge-background-color,
-      label-variables.getLabelColor(gray, bg-color),
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      height,
-      clr-badge-height,
-      variables.$clr_baselineRem_0_625,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-radius,
-      clr-badge-border-radius,
-      variables.$clr_baselineRem_0_416,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      font-size,
-      clr-badge-font-size,
-      variables.$clr_baselineRem_0_416,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(font-weight, clr-badge-font-weight, inherit, variables.$clr-use-custom-properties);
-    @include mixins.css-var(letter-spacing, clr-badge-letter-spacing, inherit, variables.$clr-use-custom-properties);
-    @include mixins.css-var(line-height, clr-badge-line-height, normal, variables.$clr-use-custom-properties);
+    min-width: var(--clr-badge-min-width);
+    background-color: var(--clr-badge-background-color);
+    height: var(--clr-badge-height);
+    border-radius: var(--clr-badge-border-radius);
+    font-size: var(--clr-badge-font-size);
+    font-weight: var(--clr-badge-font-weight);
+    letter-spacing: var(--clr-badge-letter-spacing);
+    line-height: var(--clr-badge-line-height);
 
-    @include mixins.css-var(
-      padding,
-      clr-badge-padding,
-      0 variables.$clr_baselineRem_4px,
-      variables.$clr-use-custom-properties
-    );
-    margin-right: variables.$clr_baselineRem_0_25;
+    padding: var(--clr-badge-padding);
+    // TODO: Should we refactor/tokenize this?
+    margin-right: #{tokens.$cds-global-space-4};
```

Comment:
In JIRA there are several tickets about removing unnecessary margins for other components. Create similar tikcet for this component too.